### ISSUE TITLE
Support slurm 22.05.11 metadata

### ISSUE THREAD

**Issue by cvallee** [mwinokan/RichQueue#14]
JSON payload does not include "slurm" key and contents are different:

```
"meta": {
     "plugin": {
       "type": "openapi\/v0.0.38",
       "name": "Slurm OpenAPI v0.0.38"
     },
     "Slurm": {
       "version": {
         "major": 22,
         "micro": 11,
         "minor": 5
       },
       "release": "22.05.11"
     }
   },

```

**Comment by mwinokan** [comment]
@cvallee can you please try again but with the github version of richqueue?

```
git clone git@github.com:mwinokan/RichQueue
cd RichQueue
python -m richqueue.main
```

If it works I'll push to PyPI

**Comment by cvallee** [comment]
Just tried and it works 👍

**Comment by mwinokan** [comment]
fix published to PyPI in version 1.6
